tools/gcp_init/installer.py

```diff
@@ -0,0 +1,713 @@
+#!/usr/bin/env python
+
+"""installer automates config GCE and compute instances for turbinia."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse as argp
+import imp
+import json
+import logging as log
+import os
+import shutil
+import sys
+import tempfile as tempf
+import time
+import traceback
+
+import fabric.api as fab
+import fabric.contrib.project as fabp
+from fabric.exceptions import NetworkError as FabNetworkError
+
+CONFIG_FILE = '../../turbinia/config/turbinia_config.py'
+tcfg = imp.load_source('turbiniaconfig', CONFIG_FILE)
+
+try:
+  if tcfg.UNCONFIGURED:
+    log.fatal('Please customize turbinia\'s configuration file'
+              'for your site: {0:s}'.format(CONFIG_FILE))
+except AttributeError:
+  pass
+
+fab.env.use_ssh_config = True
+
+
+def Init():
+  """Initialize options for this install run.
+
+  Returns:
+    opts (argparse namespace class): argparse data structure containing options
+  """
+  p = argp.ArgumentParser()
+  p.add_argument('--verbose', help='enable verbose output',
+                 action='store_true', default=False)
+  p.add_argument('--debug', help='enable debug messages',
+                 action='store_true', default=False)
+  p.add_argument('--tmpdir', help='set temporary directory',
+                 default=tempf.mkdtemp(dir='/var/tmp', prefix='turbt-'))
+  p.add_argument('--cache', help='set cache directory',
+                 default=tempf.mkdtemp(dir='/var/tmp', prefix='turbc-'))
+  p.add_argument('--no_prune_tc', help='don\'t prune cache and tmp directories',
+                 action='store_true', default=False)
+  p.add_argument('--no_build', help='skip wheel build',
+                 action='store_true', default=False)
+  p.add_argument('--troot', help='set local turbinia root directory',
+                 default='../../../turbinia')
+  p.add_argument('--tdir', help='set remote root directory',
+                 default='/usr/local/turbinia')
```

Comment:
I think we are using the /opt/ prefix in other locations.   We should make sure they're the same.   I think this is good for now; I'll talk to beamcodeup@ tomorrow about the other locations.

Reply:
Actually, I think I like this directory better, and beamecodeup@ seemed good with it too.